<launch>

    <!-- Actual node launch-->
    <node pkg="ros_awsiot_thingshadow" name="sync" type="sync.py" output="screen" required="true">

        <param name="host" value="XXXXXXXXXXXXX-ats.iot.ap-northeast-1.amazonaws.com" />
        <param name="thingName" value="ros" />
        <param name="certificatePath" value="$(HOME)/.awsiot/certificate.pem.crt"/>
        <param name="privateKeyPath"  value="$(HOME)/.awsiot/private.pem.key"/>
        <param name="rootCAPath"      value="$(HOME)/.awsiot/AmazonRootCA1.pem"/>

        <param name="dyn_reconf_args" value="$(find ros_awsiot_thingshadow)/config/sample-config.yaml"/>

    </node>

</launch>
